### PR TITLE
Add matrix labels for plot_multisubj example

### PR DESCRIPTION
**Pull request by emdupre**
In response to #1568, adds atlas labels to generated matrices in `plot_multi_subject_connectome`
example for ease of interpretability.

### CODE REVIEW

**Comment by bthirion** [comment]
LGTM, thx.

File: examples/03_connectivity/plot_multi_subject_connectome.py [hunk 1]
```diff
@@ -92,6 +94,8 @@ def plot_matrices(cov, prec, title):
 # -------------------
 atlas_imgs = image.iter_img(msdl_atlas_dataset.maps)
 atlas_region_coords = [plotting.find_xyz_cut_coords(img) for img in atlas_imgs]
+labels = msdl_atlas_dataset.labels
+labels = [label.decode("utf-8") for label in labels]
```
Comment (GaelVaroquaux) [inline examples/03_connectivity/plot_multi_subject_connectome.py]: That's ugly :$. Do you think that we should be doing the decoding in the fetch_atlas_msdl function?

**Comment by GaelVaroquaux** [comment]
Overall +1, but it would be useful to move the ".decode" to the fetcher function, so as not to distract our users with it, unless you see a drawback of doing that.

**Comment by GaelVaroquaux** [comment]
Overall +1, but it would be useful to move the ".decode" to the fetcher function, so as not to distract our users with it, unless you see a drawback of doing that.

**Comment by emdupre** [comment]
I think moving the decoding to the fetcher function is a great idea! The latest commit should do that.

**Comment by GaelVaroquaux** [comment]
LGTM. Thanks! Merging!

**Pull request merged**